client/utils/account-fees.tsx

```diff
@@ -232,10 +245,11 @@ export const formatAccountFeesDescription = (
 		// TODO: Figure out how the UI should work if there are several "discount" fees stacked.
 		let percentage, fixed;
 
-		if ( currentFee.discount ) {
+		if ( 'discount' in currentFee ) {
```

Comment:
Looks not good, why not keep the original construct?

Reply:
@kalessil the TS compiler did not like the original construct, it was complaining that it could be `undefined`. I'll try to find a solution

Reply:
Hm, then the re is an incomplete definition somewhere on the way there.

Reply:
I could not find a different way of doing this.. The way I see it: `currentFee` can be of two different types `BaseFee` or `DiscountFee`. Since `BaseFee` does not have the `discount` property, the compiler complaints. 

I am afraid that I reached a dead end here...To be honest, I believe it is fair to check if the property exists in this way (it is covered in TS docs [here](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#the-in-operator-narrowing) but please let me know if there is a better way.

Reply:
At first, I had the same reaction as @kalessil. But I think I see the point now.

Reply:
Addressed in c17fb6880af211def0bbe8d6def9bff350ec514c